CHANGES.md

```diff
@@ -33,6 +33,7 @@
   + This new implementation does not depend on conduit and has a simpler and
     more flexible API
 - async: Adapt cohttp-curl-async to work with core_unix.
+- async: Add upper bound to core_kernel when testing to fix the CI
```

Comment:
Do we usually include test related changes?

Reply:
Sometimes we did, sometimes not.
In any case I just noticed that this is fixed by #633

Reply:
I don't know if this PR is still needed, I'll try to test locally first

Reply:
I guess I'm more used to thinking about the change log as something user facing. So I would prefer to leave uninteresting information out.

Reply:
This was user-facing though, since if you install with or without tests, it changes the dependencies